themes/Flex/templates/base.html

```diff
@@ -89,14 +89,14 @@
   {% endif %}
   <aside>
     <div>
-      <a href="{{ SITEURL }}">
+      <a href="{{ SITEURL }}" aria-label="{{ SITETITLE }}">
         {% if SITELOGO %}
         <img src="{{ SITELOGO }}" alt="{{ SITETITLE }}" title="{{ SITETITLE }}">
         {% else %}
         <img src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/img/profile.png" alt="{{ SITETITLE }}" title="{{ SITETITLE }}">
         {% endif %}
       </a>
-      <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
+      <h1><a href="{{ SITEURL }}" aria-label="{{ SITETITLE }}">{{ SITETITLE }}</a></h1>
```

Comment:
Since this link is the same as the above, can we just remove it and leave the text?

Reply:
Ohhhhhh ok. So I think what's going on in this block is the template is making the sidebar logo clickable, but then we don't have SITETITLE defined anywhere so that h1 link is reeeeeeeeeally empty. We also listed that same link again in `content/pages/home.md` (both files in that `pages` folder show up as links in the sidebar), which is where the `clepy` link that shows up in the sidebar comes from. So none of those {{ SITETITLE }} calls do anything! Goooooooooot it.